vertexai/index.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<!--
+Copyright (c) 2016 Google Inc.
```

Comment:
2024

Reply:
Done, wow the quickstart repo is really old.